.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,22 @@
-[Fill in the placeholders below. Delete any headings and placeholders that you do not fill in. Delete this line as well]
+**Operating System:** 
+**Commit/Build:** [e.g. 7b421c8 It can be found at the left bottom corner of the main menu.]
 
-**OS:** [e.g. Windows 10]
-**Version:** [e.g. 0.0.5]
-**Commit/Build:** [e.g. 426e106]
+**Explanation of the issue:**
 
-[Explanation of the issue...]
-
-- [ ] Reproducible in RCT2 (vanilla)?
-- [ ] Multiplayer?
+**Reproducible in RCT2 (vanilla)?:**
+**Multiplayer affected?:**
```

Comment:
I think the intention behind the original "Multiplayer affected?" is to find bugs, that only occur in multiplayer. Because it's quite new and experimental and buggy. "Affects multiplayer?" sounds to me like more "Occurs in multiplayer?" which is rather useless because pretty much every singleplayer bug will also happen in multiplayer.